book/test-utils/envir_hook.qmd

```diff
@@ -20,6 +20,14 @@ knitr::opts_template$set(
   ),
   skip_if_testing = list(
     eval = isFALSE(testthat::is_checking() || testthat::is_testing())
+  ),
+  app = list(
+    webshot = "webshot2",
+    screenshot.force = TRUE,
+    screenshot.opts = list(delay = 3),
```

Comment:
When testing locally I have noticed noticed that `webshot2` is much faster than `webshot`. This can be of course overwritten on individual chunk level so this value should be sort of a default value - also applicable for the ones using this template w/o overwrites. 
The thing is that I am unable to test it properly. Locally I have everything with value being 1.5 secs and for GHA I have increased it x2.

Reply:
OK

```suggestion
    screenshot.opts = list(delay = 5),
```